apps/authentication-service/cypress/support/commands.ts

```diff
@@ -36,3 +38,63 @@ import "@testing-library/cypress/add-commands";
 //     }
 //   }
 // }
+
+Cypress.Commands.add("loginM2MAuth0", () => {
+  const client_id = Cypress.env("AUTH0_CLIENT_ID") as string;
+  const client_secret = Cypress.env("AUTH0_CLIENT_SECRET") as string;
+  const audience = Cypress.env("AUTH0_AUDIENCE") as string;
+
+  cy.request({
+    method: "POST",
+    url: `https://${Cypress.env("AUTH0_DOMAIN")}/oauth/token`,
+    body: {
+      grant_type: "client_credentials",
+      audience,
+      client_id,
+      client_secret,
+    },
+  }).then((response: Cypress.Response<JwtPayload>) => {
+    cy.window().then((win: Cypress.AUTWindow) =>
+      win.localStorage.setItem("jwt-m2m", response.body.access_token as string),
+    );
+  });
+});
+
+Cypress.Commands.add("loginClientAuth0", () => {
```

Comment:
You can remove all the loginClientAuth0 stuff now right? We're not using the m2m token?